api/src/services/roles.ts

```diff
@@ -146,7 +147,45 @@ export class RolesService extends ItemsService {
 		return;
 	}
 
-	override async updateOne(key: PrimaryKey, data: Record<string, any>, opts?: MutationOptions): Promise<PrimaryKey> {
+	private isIpAccessValid(value?: any[] | null): boolean {
+		if (!value || !Array.isArray(value)) return false;
```

Comment:
I think `null` is a valid value for `ip_access`

```suggestion
		if (value === null) return true;

		if (!Array.isArray(value)) return false;
```

Reply:
Youre right!! :heart: Thx, fixed.